File: BaekJoon/Bronze/1978/Python/1978.py
import sys

def isPrime(n):
    if n == 1:
        return False
    for i in range(2, num):
        if num % i == 0:
            return False
    return True

n = int(sys.stdin.readline())
arr1 = map(int, sys.stdin.readline().split())
cnt = 0
for num in arr1:
    if isPrime(num):
        cnt += 1
print(cnt)

File: BaekJoon/Bronze/1978/C++/1978.cpp
// no.1978: 소수 찾기 (B2)

#include <cstdio>
#include <sstream>
using namespace std;

bool isPrime(int num);

int main() {
    int n;
    scanf("%d", &n);
    char str[10000];
    getchar();
    fgets(str, 10000, stdin);
    stringstream ss(str);
    string tmp;
    int count = 0;
    while(ss >> tmp) {
        if(isPrime(stoi(tmp)))
            count++;
    }
    printf("%d\n", count);
    return 0;
}

bool isPrime(int num) {
    if(num == 1)
        return false;
    for(int i=2; i<num; i++) {
        if(num%i == 0) {
            return false;
        }
    }
    return true;
}